server/routes/add-any-alert/view.njk

```diff
@@ -0,0 +1,120 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "moj/components/date-picker/macro.njk" import mojDatePicker %}
+{% extends "partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - Create alert code" %}
+{% set mainClasses = "app-container govuk-body govuk-!-static-padding-top-0" %}
+{% set backLinkLabel = "Manage prisoner alerts" %}
+{% set backUrl = "/" %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <h1 id="addCaseNote">Add an alert</h1>
+        </div>
+    </div>
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form method="post" novalidate="novalidate" autocomplete="off">
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+                {{ govukCharacterCount({
```

Comment:
I don't think there's a requirement for a character count in the ticket. Looks a bit strange for a value that's always 7 characters